gstools/field/base.py

```diff
@@ -343,6 +470,134 @@ def plot(
 
         return r
 
+    def set_pos(self, pos, mesh_type="unstructured", info=False):
+        """
+        Set positions and mesh_type.
+
+        Parameters
+        ----------
+        pos : :any:`iterable`
+            the position tuple, containing main direction and transversal
+            directions
+        mesh_type : :class:`str`, optional
+            'structured' / 'unstructured'
+            Default: `"unstructured"`
+        info : :class:`bool`, optional
+            Whether to return information
+
+        Returns
+        -------
+        info : :class:`dict`
+            Information about settings.
```

Comment:
I think a warning or note field in the docs is appropriate here, to warn users that the current fields will get overwritten.

Reply:
Done.